FirebasePhoneAuthUI/FUIPhoneEntryViewController.h

```diff
@@ -40,25 +42,29 @@ __attribute__((deprecated("This is deprecated API and will be removed in a futur
 __attribute__((deprecated("This is deprecated API and will be removed in a future release."
                           "Please use initWithNibName:bundle:authUI:phoneNumber:")));
 
-/** @fn initWithAuthUI:phoneNumber:
+/** @fn initWithAuthUI:phoneNumber:countryCodes:
     @brief Convenience initializer.
     @param authUI The @c FUIAuth instance that manages this view controller.
     @param phoneNumber The phone number which is being verifying.
+    @param countryCodes Available country codes For the view controller to use.
  */
 - (instancetype)initWithAuthUI:(FUIAuth *)authUI
-                   phoneNumber:(nullable NSString *)phoneNumber;
+                   phoneNumber:(nullable NSString *)phoneNumber
+                  countryCodes:(nullable FUICountryCodes *)countryCodes;
 
-/** @fn initWithNibName:bundle:authUI:phoneNumber:
+/** @fn initWithNibName:bundle:authUI:phoneNumber:countryCodes:
     @brief Designated initializer.
     @param nibNameOrNil The name of the nib file to associate with the view controller.
     @param nibBundleOrNil The bundle in which to search for the nib file.
     @param authUI The @c FUIAuth instance that manages this view controller.
     @param phoneNumber The phone number which is being verifying.
+    @param countryCodes Available country codes For the view controller to use.
  */
 - (instancetype)initWithNibName:(nullable NSString *)nibNameOrNil
                          bundle:(nullable NSBundle *)nibBundleOrNil
                          authUI:(FUIAuth *)authUI
```

Comment:
API nit: `authUI` can also be made nullable, and `[FUIAuth defaultAuthUI]` can be used if nil is specified.

This also isn't properly part of your change, but reduces some API awkwardness. I can make this change separately.

Reply:
I think doing it in a separate CL is better --- we can have a chance to make every FUIAuth in all auth controller nullable. It's better to be done at FUIAuthBaseViewController level.